components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java

```diff
@@ -143,7 +207,7 @@ public boolean insertWorkerConfiguration(WorkerConfigurationDetails workerConfig
         try {
             return this.insert(columnNames, records, WORKER_CONFIG_TABLE);
         } catch (RDBMSTableException e) {
-            throw  new RDBMSTableException(e.getMessage(),e);
+            throw new RDBMSTableException(e.getMessage(), e);
```

Comment:
Instead of `e.getMessage()`, put a descriptive, context aware error message.

Reply:
Fixed in acaafe89c43697be9ef1532a725f8c6bdc4ae6d6